medusa/metadata/wdtv.py

```diff
@@ -162,12 +150,11 @@ def get_season_poster_path(show_obj, season):
                 break
 
         if not season_dir:
-            logger.log(u'Unable to find a season directory for season {season_num}'.format
-                       (season_num=season), logger.DEBUG)
+            log.debug(u'Unable to find a season directory for season {0}', season)
             return None
 
-        logger.log(u'Using {path}/folder.jpg as season dir for season {season_num}'.format
-                   (path=season_dir, season_num=season), logger.DEBUG)
+        log.debug(u'Using {path}/folder.jpg as season dir for season {x}',
+                  {'path': season_dir, 'x': season})
```

Comment:
My main reasoning was for locations like the seeders example where you can't use `number` since there's more than one number involved:

Compare:
'Downloading {name} with {seeders} seeders and {leechers} leechers from {provider}'
'Downloading {name} with {x} seeders and {y} leechers from {provider}'

I think anyone reading that without any other context would understand that `x` is the number of seeders and `y` is the number of leechers.